pubsub/pubsub.go

```diff
@@ -192,37 +192,17 @@ func (t *Topic) Shutdown(ctx context.Context) (err error) {
 }
 
 // As converts i to provider-specific types.
-//
-// This function (and the other As functions in this package) are inherently
-// provider-specific, and using them will make that part of your application
-// non-portable, so use with care.
-//
-// See the documentation for the subpackage used to instantiate Bucket to see
-// which type(s) are supported.
-//
-// Usage:
-//
-// 1. Declare a variable of the provider-specific type you want to access.
-//
-// 2. Pass a pointer to it to As.
-//
-// 3. If the type is supported, As will return true and copy the
-// provider-specific type into your variable. Otherwise, it will return false.
-//
-// Provider-specific types that are intended to be mutable will be exposed
-// as a pointer to the underlying type.
-//
-// See
-// https://github.com/google/go-cloud/blob/master/internal/docs/design.md#as
-// for more background.
+// See https://godoc.org/gocloud.dev#As for background information, the "As"
+// examples in this package for examples, and the provider-specific package
+// documentation for the specific types supported for that provider.
 func (t *Topic) As(i interface{}) bool {
 	return t.driver.As(i)
 }
 
-// ErrorAs converts err to provider-specific types.
-// ErrorAs panics if target is nil or not a pointer.
+// ErrorAs converts i to provider-specific types.
+// ErrorAs panics if i is nil or not a pointer.
 // ErrorAs returns false if err == nil.
-// See Topic.As for more details.
+// See https://godoc.org/gocloud.dev#As for background information.
```

Comment:
Same comment as above.

Reply:
Ugh... our naming of parameters in ErrorAs is inconsistent.

i here: https://github.com/google/go-cloud/blob/master/blob/blob.go#L500
target here: https://github.com/google/go-cloud/blob/master/pubsub/pubsub.go#L222

I'll fix the inconsistency in a separate PR